speech-commands/src/dataset.ts

```diff
@@ -473,10 +473,16 @@ export class Dataset {
         // for tf.data currently. Tighten the types when the tf.data bug is
         // fixed.
         // tslint:disable:no-any
-        const xTrain =
-            tf.data.array(trainXs as any).map(x => tf.tensor3d(x as any, [
-              numFrames, uniqueFrameSize, 1
-            ]));
+        console.log('Calling map()');        // DEBUG
+        console.log('trainXs = ', trainXs);  // DEBUG
+        const xTrain = tf.data.array(trainXs as any).map(x => {
+          console.log(
+              `In map: numFrames=${numFrames}; ` +
+                  `uniqueFrameSize=${uniqueFrameSize}; ` +
+                  `x = `,
+              x);  // DEBUG
```

Comment:
@kangyizhang I got some debugging findings: 

In tfjs 1.1.2, `x` here is 

x =  Float32Array [
  -1.705195665359497,
  -1.6510624885559082,
  -1.5969291925430298,
  -1.542796015739441,
...

But in tfjs 1.2.9, it becomes 

In map: numFrames=4; uniqueFrameSize=16; x =  { '0': -1.705195665359497,
  '1': -1.6510624885559082,
  '2': -1.5969291925430298,
  '3': -1.542796015739441,
  '4': -1.488662838935852,

This seems to be related to some changes in tfjs-data.